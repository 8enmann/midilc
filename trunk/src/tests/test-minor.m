/* Test minor
   Tests creating a minor chord from its root.  Library function. */

main(){
  Sequence s;
  Chord c;
  s = new_sequence();
  
  c = minor(C);
  s = s + c;
  set_tempo(30);
  play(s);
}

minor(root)
{
	Chord c;
	
	c = new_chord(root);
	
	c = c + (root .+ 3);
	c = c + (root .+ 7);
	return c;
}
