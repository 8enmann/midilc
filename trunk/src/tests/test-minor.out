Tempo,30
0,4,60
0,4,63
0,4,67

